src/style/theme.ts

```diff
@@ -0,0 +1,19 @@
+import { createTheme, Theme } from "@mui/material";
```

Comment:
this is a placeholder file for now for common control styles to be merged with particular app related styles/palette